code/src/Components/Select.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { Progress } from 'Components/Progress'
+
+export const Select = ({ food, setFood }) => {
+    return (
+        <form>
+            <Progress current='3' total='3'/> 
+            <h2>Here we go: select your favourite sweet from the dropdown! Oops! You can choose only one Sweetie!</h2>
```

Comment:
More of a "UX Writing" comment, but the "Oops!" text made me think I had done something wrong when I read it first 🙀

Reply:
Ooo, I see your point, yes could be. In Hungary it has totally different meaning but I am going to changes it as well! Thank you Axel!